### PR TITLE
Chap28

### PR DESCRIPTION
**Pull request by codinglikeagirl42**

### CODE REVIEW

File: src/chapters/tableau-part4/studio.rst [hunk 1]
```diff
@@ -1,2 +1,110 @@
...
+   
+| You are not limited to these topics.  If you discover something more interesting, explore away!  
+
+Setting Up the Studio:
+^^^^^^^^^^^^^^^^^^^^^^
+
+Download `FIE Fencing Womens Foil data set <https://www.kaggle.com/datasets/amichaelsen/fie-fencing-womens-foil-data>`__. There are 4 csv files. To download all csv files at once, click on the "Download" bubble next to the "New Notebook" bubble.  An orange arrow is pointing to the "Download" bubble in the image below.
+
+.. figure:: figures/download-multi-files.png
+   :alt: An orange arrow points to the "Download" bubble, highlighting the Download option of kaggle.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:32]: We have gone over this before with students so why repeat it?

File: src/chapters/tableau-part4/studio.rst [hunk 2]
```diff
@@ -1,2 +1,110 @@
...
+#. Here is more information about the `International Fencing Federation <https://fie.org/>`__.
+
+
+
+EDA
+^^^
+
+1. There are 4 csv's.
+   
+   A. Work with the ones you think you will need to answer your questions.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:48]:
```suggestion
1. The dataset contains 4 CSVs. You only need to work with the ones that will provide answers to your questions.
```

File: src/chapters/tableau-part4/studio.rst [hunk 3]
```diff
@@ -1,2 +1,110 @@
...
+
+#. Do you need to clean any of the data you have decided to work with?
+
+   A. Any nulls?  
+
+      1. Are they appropriate for the data?
+
+   #. Any discrepancies?
+
+      1.Hint: ages might need some attention.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:69]: This formatting is off. The list items are not lining up. Please fix this.

File: src/chapters/tableau-part4/studio.rst [hunk 4]
```diff
@@ -1,2 +1,110 @@
...
+
+Manipulation
+^^^^^^^^^^^^
+
+#. Do you need to use any calculations to answer the questions?
+#. Thinking of your dashboard, how do you think you would like to present your data to your audience? 
+#. In order to do this, do you need:
+
+   A. filters?
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:78]:
```suggestion
   A. Filters? 
```

File: src/chapters/tableau-part4/studio.rst [hunk 5]
```diff
@@ -1,2 +1,110 @@
...
+^^^^^^^^^^^^^^^^^
+
+| Before you turn in your vizzes:
+
+#. Make sure that they are easy to read. 
+#. Review and edit any axes so that they don't contain any file information. 
+#. Make sure any filtering, group, or set information is easy to understand. 
+#. If you want to explore fonts and colors, go right ahead. 
+
+   A. Feel free to change the colors of any/all of your charts.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:99]: These two items are basically the same. Leave off the subpoint.

**Review by gildedgardenia — CHANGES_REQUESTED** [review]
Please review the formatting and grammar on every page and address any additional changes that need to be made.

File: src/chapters/tableau-part4/studio.rst (hunk 1 above)
Comment (codinglikeagirl42) [reply src/chapters/tableau-part4/studio.rst]: We have repeated it in other chapters.  Originally done in canvas to help students.

File: src/chapters/tableau-part4/studio.rst [hunk 6]
```diff
@@ -1,2 +1,107 @@
...
+   #. Ages
+   #. Location and/or Time Zone
+   #. What factors predict wins? 
+   
+You are not limited to these topics.  If you discover something more interesting, explore away!  
+
+Setting Up the Studio:
+^^^^^^^^^^^^^^^^^^^^^^
+
+Download `FIE Fencing Womens Foil data set <https://www.kaggle.com/datasets/amichaelsen/fie-fencing-womens-foil-data>`__. There are 4 csv files. To download all csv files at once, click on the ``Download`` bubble next to the ``New Notebook`` bubble.  An orange arrow is pointing to the ``Download`` bubble in the image below.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:29]:
```suggestion
Download `FIE Fencing Womens Foil data set <https://www.kaggle.com/datasets/amichaelsen/fie-fencing-womens-foil-data>`__. There are 4 CSV files. To download all CSV files at once, click on the ``Download`` bubble next to the ``New Notebook`` bubble.  An orange arrow is pointing to the ``Download`` bubble in the image below.
```

File: src/chapters/tableau-part4/studio.rst [hunk 7]
```diff
@@ -1,2 +1,107 @@
...
+
+.. figure:: figures/download-multi-files.png
+   :alt: An orange arrow points to the ``Download`` bubble, highlighting the ``Download`` option of kaggle.
+
+#. Open the csv file in Tableau Public.
```
Comment (gildedgardenia) [inline src/chapters/tableau-part4/studio.rst:34]:
```suggestion
#. Open the CSV file in Tableau Public.  
```

**Review by gildedgardenia — APPROVED** [review]
Looks good!

**Pull request merged**